core/validatorapi/eth2types.go

```diff
@@ -135,3 +135,41 @@ func (v v1Validator) MarshalJSON() ([]byte, error) {
 
 	return bytes.ToLower(b), nil // ValidatorState must be lower case.
 }
+
+// syncCommitteeDutiesRequest defines the request to the getSyncCommitteeDuties endpoint.
+// See https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/getSyncCommitteeDuties.
+type syncCommitteeDutiesRequest []eth2p0.ValidatorIndex
```

Comment:
rather just rename `attesterDutiesRequest` to `valIndexesJSON` and re-use that type.

Reply:
done